django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -456,6 +456,13 @@ input[type="submit"], button {
     pre.literal-block {
         overflow: auto;
     }
+
+        
+    .submit-row input{
+        margin-top: 5px !important;
+        margin-bottom: 5px !important;
+    }
+
```

Comment:
IMO, `!important` is not necessary.

What do you think about decreasing `padding` instead? For example:
```css
.submit-row input {
  padding: 5px 10px;
}
```

Reply:
thnkyou sir for review, reduced padding is also not working. and idk why the by removing !important the changes is not reflecting.

Reply:
It works for me :thinking: 
Before:
![image](https://user-images.githubusercontent.com/2865885/151344105-f200502c-fd69-4ca9-a0fd-9cdbf90e2a08.png)

After:
![image](https://user-images.githubusercontent.com/2865885/151343955-46814a7d-f37c-4af0-bd69-5f34ac3a63e9.png)

Reply:
but sir its not working if we have a delete btn 
![image](https://user-images.githubusercontent.com/68425016/151345017-691556b9-bf80-49a5-b752-1fead5b8f222.png)

Reply:
True, I'd play with `submit-row` and `deletelink` `margin` and `padding` attributes to keep them in a single line.

Reply:
ok understood , it should be in single line. i revert all changes and again resolve it as you said